implants/lib/c2/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "c2"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tonic = { workspace = true }
+prost = "0.12"
+prost-types = { workspace = true }
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
```

Comment:
Can we use the version the parent workspace is using.

Reply:
o7